pubsub/envelope.go

```diff
@@ -134,6 +136,7 @@ func FromCloudEvent(cloudEvent []byte, topic, pubsub, traceParent string, traceS
 	m[TraceStateField] = traceState
 	m[TopicField] = topic
 	m[PubsubField] = pubsub
+	m[TimeField] = publishTime
```

Comment:
I think we can check here if TimeField is already present since unmarshal has happened already on line 129.

Reply:
Yes, good catch. I'll fix it and repush !